zanata-common-api/pom.xml

```diff
@@ -236,7 +243,26 @@
     <dependency>
       <groupId>org.codehaus.enunciate</groupId>
       <artifactId>enunciate-jersey-rt</artifactId>
+      <version>${enunciate.version}</version>
       <scope>provided</scope>
+      <exclusions>
```

Comment:
From memory, I think I copied the exclusions from zanata-parent. The idea is to eliminate client-only dependencies like Enunciate from zanata-parent: once that happens we won't be able to inherit the exclusions any more.

The usual reason for bans/exclusions in Zanata is that the banned jars duplicate the classes found in another jar.